### DISCUSSION TITLE
Option to remove workspace icon

### DISCUSSION THREAD

**Discussion by 10Electra** [zen-browser/desktop#3251] [Ideas]
I don't want to represent each workspace with an emoji - I would rather use the first letter (or better, first two letters) of the workspace name to recognise it.

Doesn't seem like there's an option to do this deliberately (have got it to happen as its a bit buggy with adding icons).

**Comment by pkolyvas** [comment]
I came here to ask for the same thing. On my macOS Zen, the workspaces are letters. On Linux (KDE Plasma 6 using the version in pacman) it seems I have no choice but to use icons. I agree with @10Electra - this is a suboptimal experience. 

I'd prefer to not select an icon and, if no icon is selected, the workspace defaults to the first letter of the workspace name.

**Comment by rpoovey** [comment]
This used to be how it functioned. Having this back would be so much nicer. 

For example, I work in IT and I use a workspace per client. Having the first letter of their company name as the workspace icon makes it easy for me to see which one I need to switch to. Representing a client name with an emoji makes no sense. Prior to version 1.10b the workspaces I had setup long ago stayed, and all new workspaces had to have emojis. Now with 1.10b, those letters turned into dots so its extra confusing. 

Please fix this.
